src/pages/MusicView.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useContext, useEffect } from "react";
+import AudioPlayer from "react-h5-audio-player";
+import "react-h5-audio-player/lib/styles.css";
+import { useHistory } from "react-router-dom";
+import Navbar from "../components/Navbar";
+import TaskList from "../components/global/TaskList";
+import { fetchTasks } from "../services/databaseService";
+import { AuthContext } from "../services/providers/authProvider";
+import { CurrentTaskContext } from "../components/timer-modal/TimerContextProvider";
+
+const playlist = [
+  {
+    name: "Nice piano and ukulele",
+    author: "Royalty",
+    img: "https://www.bensound.com/bensound-img/buddy.jpg",
+    audio: "https://www.bensound.com/bensound-music/bensound-buddy.mp3",
+    duration: "2:02",
+  },
+  {
+    name: "Gentle acoustic",
+    author: "Acoustic",
+    img: "https://www.bensound.com/bensound-img/sunny.jpg",
+    audio: "https://www.bensound.com//bensound-music/bensound-sunny.mp3",
```

Comment:
I see that this is royalty-free music which is great :) However, please check if we have to attribute/credit the author in the readme or something similar.